code/modules/mob_spawn/ghost_roles/spider_roles.dm

```diff
@@ -134,6 +137,17 @@
 	var/datum/antagonist/spider/spider_antag = new granted_datum(directive)
 	spawned_mob.mind.add_antag_datum(spider_antag)
 
+/obj/effect/mob_spawn/ghost_role/spider/abnormal
+	name = "abnormal egg cluster"
+	color = rgb(0, 148, 211)
+	you_are_text = "You are an enriched spider."
```

Comment:
This doesn't seem meaningful to the player, I don't see a strong reason for the egg type to be noted in this spawn message.

Reply:
minor error woops